### DISCUSSION TITLE
ESP32 C6 as Zigbee gateway

### DISCUSSION THREAD

**Discussion by nouhin** [arendst/Tasmota#22217] [Q&A]
Hello everyone,

I'm relatively new to IoT and home automation and am exploring DIY solutions for integrating devices into my smart home setup. I'm interested in using an ESP32-C6 board as a Zigbee gateway. Specifically, I would like to know if it's feasible to configure a single ESP32-C6 board to act as a Zigbee gateway that is fully compatible with the Zigbee Home Automation (ZHA) integration in Home Assistant.

Additionally, I'm curious about any necessary firmware, libraries, or configurations required to achieve this functionality. Are there specific challenges or limitations I should be aware of when using an ESP32-C6 for this purpose?

Thank you in advance for your insights and guidance!

**Comment by s-hadinger** [comment]
As per esp-idf documentation, there is a single radio stage on C6 that can be either wifi or zigbee/thread but not both at the same time. Switching from one to the other over time is not a viable option. 
So the answer is no, you need at least two chips for wifi + zigbee

**Reply by nikito7** [reply]
Why not with ethernet?

Tasmota have some issues without wifi. But should work.

**Reply by s-hadinger** [reply]
On paper it should work, but this limits even more the potential hardware.

I don't see any manufacturer offering an ethernet-only device. Not having wifi would be seen as a major problem. It's much cheaper to add a second MCU than an ethernet port.
